CMake/Dependencies.cmake

```diff
@@ -291,8 +291,15 @@ if(NOT ANDROID AND NOT EMSCRIPTEN)
   macro_log_feature(SDL2_FOUND "SDL2" "Simple DirectMedia Library needed for input handling in samples" "https://www.libsdl.org/" FALSE "" "")
   if(SDL2_FOUND AND WIN32 AND NOT SDL2_BINARY)
     # fix linking static SDL2 on windows
-    set(SDL2_LIBRARY ${SDL2_LIBRARY} winmm.lib imm32.lib version.lib)
+    set(SDL2_LIBRARIES ${SDL2_LIBRARIES} winmm.lib imm32.lib version.lib)
   endif()
+  if(SDL2_FOUND AND NOT TARGET SDL2::SDL2)
+    add_library(SDL2::SDL2 INTERFACE IMPORTED)
```

Comment:
now you can get rid of SDL2_INCLUDE_DIRS and SDL2_LIBRARIES elsewhere altogether